client/payment-details/dispute-details/dispute-notice.tsx

```diff
@@ -70,6 +84,13 @@ const DisputeNotice: React.FC< DisputeNoticeProps > = ( {
 							href={ learnMoreDocsUrl }
 						/>
 					),
+					link: (
+						<Link
+							href={ learnMoreDocsUrl }
+							target="_blank"
+							type="external"
```

Comment:
We can add a `noopener` and `noreferrer` for external links as a security measure. Although modern browsers automatically apply these for `target="_blank" links`, [MDN still recommend it](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#security_and_privacy) and it remains common practice in this codebase.

```suggestion
							type="external"
							rel="noopener noreferrer"
```

Reply:
Fixed automatically when I switched back to `ExternalLink` 👍